libs/ddhub-client-gateway-frontend/ui/channels/src/lib/containers/modals/Details/Topics/Topics.tsx

```diff
@@ -11,36 +11,71 @@ interface TopicsProps {
 
 export const Topics: FC<TopicsProps> = ({ topics }) => {
   const { classes } = useStyles();
-  const { openTopicDetails } = useTopicsEffects();
+  const {
+    openTopicDetails,
+    rowsPerPage,
+    page,
+    handleChangePage,
+    handleChangeRowsPerPage,
+  } = useTopicsEffects();
+
+  const rows = [];
```

Comment:
What do you think about something like this? It's more or less.
```suggestion
  const rows = topics.slice(startIdx, endIdx).map(row => <Box...);
```
However `endId` should calculated a little bit different. It should be the smaller value between `startIdx + rowsPerPage` and `topics.length`. But it's up to you if you want to go this way.

Reply:
Agreed, this is a neater solution. I updated the code, please re-review. Thanks!